File: JS/Math/FizzBuzz.js
module.exports = {
 //param A : integer
 //return a array of strings
	fizzBuzz : function(A){
        var res = [];
        for(var val = 1; val <= A; val++){
            var here = "";
            if(val % 3 != 0 && val % 5 != 0)here = val;
            else{
                if(val % 3 == 0)here += "Fizz";
                if(val % 5 == 0)here += "Buzz";
            }
            res.push(here);
        }
        return res;
	}
};

File: Java/Math/FizzBuzz.java
public class Solution {
    public ArrayList<String> fizzBuzz(int A) {
        ArrayList<String> res = new ArrayList<String>();
        for(int i = 1; i <= A; i++){
            String here = "";
            if((i % 3 != 0) && (i % 5 != 0))here += i;
            else{
            if(i % 3 == 0)here += "Fizz";
            if(i % 5 == 0)here += "Buzz";}
            res.add(here);
        }
        return res;
    }
}
